libs/designsystem/src/lib/components/list/list.component.spec.ts

```diff
@@ -141,32 +141,60 @@ describe('ListComponent', () => {
   });
 
   describe('function: ngOnInit', () => {
-    it('should enable load more, if there is a subscriber to the loadMore event emitter', () => {
-      component.loadOnDemand.subscribe((_: LoadOnDemandEvent) => {});
+    describe('there is a subscriber to the loadMore event', () => {
+      beforeEach(() => {
+        component.loadOnDemand.subscribe((_: LoadOnDemandEvent) => {});
+      });
 
-      component.ngOnInit();
+      it('should enable load more, if isLoadOnDemandEnabled is not set', () => {
+        component.isLoadOnDemandEnabled = undefined;
 
-      expect(component.isLoadOnDemandEnabled).toBeTrue();
-    });
+        component.ngOnInit();
 
-    it('should disable load more, if there is no subscriber to the loadMore event emitter', () => {
-      runNgOnChanges();
+        expect(component.isLoadOnDemandEnabled).toBeTrue();
+      });
 
-      expect(component.isLoadOnDemandEnabled).toBeFalse();
-    });
-  });
+      it('should enable load more, if isLoadOnDemandEnabled is set to true', () => {
+        component.isLoadOnDemandEnabled = undefined;
 
-  describe('function: enableLoadOnDemand', () => {
-    it('should enable load more, if called with true', () => {
-      component.enableLoadOnDemand(true);
+        component.ngOnInit();
```

Comment:
Be a good boy scout 🤞
```suggestion
        component.ngAfterViewInit();
```

Reply:
I'm not sure I understand this suggestion - the function being tested is ngOnInit 😅